executor/runtime/docker/docker.go

```diff
@@ -332,32 +320,17 @@ func setupLoggingInfra(dockerRuntime *DockerRuntime) error {
 	return nil
 }
 
-func maybeSetCFSBandwidth(cfsBandwidthMode string, cfsBandwidthPeriod uint64, c *runtimeTypes.Container, hostCfg *container.HostConfig) {
+func maybeSetCFSBandwidth(cfsBandwidthPeriod uint64, c *runtimeTypes.Container, hostCfg *container.HostConfig) {
 	cpuBurst := c.TitusInfo.GetAllowCpuBursting()
-	logEntry := log.WithField("taskID", c.TaskID).WithField("bandwidthMode", cfsBandwidthMode).WithField("cpuBurst", cpuBurst)
+	logEntry := log.WithField("taskID", c.TaskID).WithField("cpuBurst", cpuBurst)
 
 	if cpuBurst {
 		logEntry.Info("Falling back to shares since CPU bursting is enabled")
 		setShares(logEntry, c, hostCfg)
 		return
 	}
 
-	if c.Resources.CPU == 0 {
```

Comment:
Wanna just inline the setCFSBandwidth in here?

Reply:
That method still has internal fallback logic for `setNanoCpus`.  I'd prefer to leave that inside the `setCFSBandwidth` method.  I'm working on a subsequent PR to address that.